src/main/java/com/google/sps/servlets/YoutubePost.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+public class YoutubePost {
```

Comment:
Could you add a constructor to this class and assign the fields in there on creation? Then all the fields can be private.
ie.
public YoutubePost(title, content...) {
  this.title = title;
  this.content = content;
  ...
}

Then in YoutubeApi, you can create the post in one go:
YoutubePost post = new YoutubePost(title(), description() ,...)